Add Hacktoberfest section to CONTRIBUTING.md (#47)
meilisearch/meilisearch-aws: CONTRIBUTING.md

First of all, thank you for contributing to MeiliSearch! The goal of this document is to provide everything you need to know in order to contribute to MeiliSearch and its different integrations.

- [Hacktoberfest](#hacktoberfest)
- [Assumptions](#assumptions)
- [How to Contribute](#how-to-contribute)
- [Development Workflow](#development-workflow)
- [Git Guidelines](#git-guidelines)
- [Release Process (for internal team only)](#release-process-for-internal-team-only)

## Hacktoberfest

It's [Hacktoberfest month](https://blog.meilisearch.com/contribute-hacktoberfest-2021/)! 🥳

> 🚀 If your PR gets accepted it will count into your participation to Hacktoberfest!

Reviewer: ```suggestion
🚀 If your PR gets accepted it will count towards your participation in Hacktoberfest!
```

> 🚀 If your PR gets accepted it will count towards your participation in Hacktoberfest!

Reviewer: ```suggestion
🚀 If your PR gets accepted it will count into your participation to Hacktoberfest!
```

> ✅ To be accepted it has either to have been merged, approved or tagged with the `hacktoberest-accepted` label.

Reviewer: ```suggestion
✅ To be accepted it has either to have been merged, approved or tagged with the `hacktoberfest-accepted` label.
```

> ✅ To be accepted it has either to have been merged, approved or tagged with the `hacktoberfest-accepted` label.

Reviewer: ```suggestion
✅ To be accepted it has either to have been merged, approved or tagged with the `hacktoberest-accepted` label.
```
